frontend/src/APIClients/CourseAPIClient.ts

```diff
@@ -13,12 +13,14 @@ type Module = {
 
 type SerializedModule = Module & {id: string}
 
+type Modules = {[id: string]: Module}
```

Comment:
nitpick: would it make sense to call this `ModulesById`? It makes the usage much more obvious

Reply:
That works too